configs/recommended.js

```diff
@@ -36,29 +36,36 @@ module.exports = {
     jest: true
   },
   rules: {
-    'no-unused-vars': 'error',
-    'no-console': ['warn', { allow: ['warn', 'error'] }],
+    '@typescript-eslint/no-duplicate-imports': 'error',
+    '@typescript-eslint/no-extra-semi': 'error',
+    '@typescript-eslint/no-use-before-define': ['error', { 'functions': false, 'classes': false }],
+    'no-duplicate-imports': 'off',
+    'no-extra-semi': 'off',
+    'no-use-before-define': 'off',
     'eol-last': 'error',
-    'no-multi-spaces': 'error',
     'import/no-default-export': 'off',
     'import/no-unresolved': ['error', { ignore: ['^ui/.*', '^plugins/.*', 'ng_mock'] }],
+    'import/order': ['warn', {
+      groups: ['builtin', 'external', 'internal', 'parent', 'sibling', 'index'],
+      pathGroups: [
+        { pattern: '@*/*', group: 'external' }, // Scoped packages are not recognized as external without this
+      ],
+      'newlines-between': 'always-and-inside-groups'
```

Comment:
I don't think this is right? Or do we use this because of the `//@ts-ignore?
<img width="1161" alt="Screenshot 2022-05-19 at 16 43 34" src="https://user-images.githubusercontent.com/15673201/169340887-6ffe6f6b-ac2e-4c03-83bd-305e697d8f5d.png">

Reply:
It means we allow users to place newlines in groups. This usually a good idea to differentiate between `plugins` and `ui`.